conjur-oss/README.md

```diff
@@ -361,10 +370,8 @@ The following table lists the configurable parameters of the Conjur OSS chart an
 |`database.ssl.key`|PostgreSQL TLS private key, base64 encoded.|`""`|
 |`dataKey`|Conjur data key, 32 byte base-64 encoded string for data encryption.|`""`|
 |`deployment.annotations`|Annotations for Conjur deployment|`{}`|
-|`image.kubernetes.repository`|Conjur Docker image repository for Kubernetes platforms|`"cyberark/conjur"`|
-|`image.kubernetes.tag`|Conjur Docker image tag for Kubernetes platforms|`"1.11.1"`|
-|`image.openshift.repository`|Conjur Docker image repository for OpenShift platform|`"registry.connect.redhat.com/cyberark/conjur"`|
-|`image.openshift.tag`|Conjur Docker image tag for OpenShift platform|`"latest"`|
+|`image.repository`|Conjur Docker image repository|`"cyberark/conjur"`|
+|`image.tag`|Conjur Docker image tag|`"1.11.1"`|
```

Comment:
just noticing that `openshift.enabled` is not in this table

Reply:
Ooh, yes. Will add this to the table.